jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -587,6 +589,12 @@ function configure_jbpm() {
     fi
 }
 
+function configure_optaplanner() {
```

Comment:
be sure to add bats tests to test the expected output for this function as well.

Reply:
done!